src/index.js

```diff
@@ -59,7 +57,7 @@ class DomEvent {
   }
 
   run (sink, scheduler) {
-    const send = e => tryEvent(scheduler.now(), e, sink)
+    const send = e => tryEvent(scheduler.currentTime(), e, sink)
```

Comment:
We should probably eat our dog food here, and use the `currentTime` function: `import { currentTime } from '@most/scheduler'`.

Reply:
as long as its Blue Buffalo i heard its pretty tasty 😝 🐶